### DISCUSSION TITLE
how to use "ctrl+PgUp" and "ctrl+PgDn" to switch to the previous/next tab?

### DISCUSSION THREAD

**Discussion by hopezh** [wezterm/wezterm#6213] [Q&A]
### What Operating System(s) are you running on?

Windows

### Which Wayland compositor or X11 Window manager(s) are you using?

Nil

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

# What I want to do
I'd like to use "Ctrl+PgUp" and "Ctrl+PgDn" to switch between tabs, which are the same keybindings I'm using to switch tabs in web browser and that in Neovim. 

# What I tried
I tried the following config:

```lua
local wezterm = require("wezterm")
local config = wezterm.config_builder()
local act = wezterm.action*

config.keys = {
	{ key = "PgUp", mods = "CTRL", action = act.ActivateTab(-1) },
	{ key = "PgDn", mods = "CTRL", action = act.ActivateTab(1) },
}

return config
```

# The error I got

```shell
error converting Lua table to Config (Config::from_dynamic: Error processing
keys.key.key (types: Config, Key, KeyNoAction) invalid keycode PgUp: invalid
KeyCode string PgUp, expected phys:CODE physical keycode string, got:
phys:PgUp.
{
    "action": {
        "ActivateTab": -1,
    },
    "key": "PgUp",
    "mods": "CTRL",
})
stack traceback:
        [C]: in metamethod 'newindex'
        [string "C:\Users\ZHANG_JI\.wezterm.lua"]:81: in main chunk
```

# My question

**May I ask how to config "ctrl+PgUp" and "ctrl+PgDn" to switch to the previous/next tab?**

Thanks.

**Comment by bew** [comment]
In the error it says `invalid keycode PgUp`. 

The keys are called `PageUp` and `PageDown`, instead of `PgUp`/`PgDn`.

You can find the list of valid keys in this page:
https://wezfurlong.org/wezterm/config/keys.html#configuring-key-assignments
